### DISCUSSION TITLE
Test Flight beta redeem code ?

### DISCUSSION THREAD

**Discussion by OdynOOb** [subconsciousnetwork/noosphere#170] [Q&A] [answered]
Hi there,
I'm new, so I'm lost :)

Did I read that there's a Test Flight beta ?
And can I can participate ?

TIA and best regards,

**Comment by gordonbrander** [comment, marked as answer]
Hi @OdynOOb please see the answer to your same question on the issue you filed in the Subconscious repository https://github.com/subconsciousnetwork/subconscious/issues/193#issuecomment-1320906587.

Subconscious app is currently in early alpha state, and test flight invites at this stage are being kept to a small group, since the software is not yet considered stable enough for a broad rollout. However, you can still build the software in XCode yourself, if curious. To be notified when a broader TestFlight Beta rolls out, sign up here https://forms.gle/H6v5J8DJ64NGxN4A6

Also worth noting: this discussion was filed in the Noosphere repository, which is a separate project. Noosphere is a protocol for thought built on top of IPFS. You can learn more here https://github.com/subconsciousnetwork/noosphere/blob/main/design/explainer.md
